src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -191,8 +191,8 @@ def _get_storage_sync_endpoint(cloud_name):
 
 def _get_synapse_analytics_endpoint(cloud_name):
     synapse_analytics_endpoint_mapper = {
-        'AzureCloud': 'dev.azuresynapse.net',
-        'AzureChinaCloud': 'dev.azuresynapse.azure.cn'
+        'AzureCloud': '.dev.azuresynapse.net',
```

Comment:
why change this? I think there's a param addDot=True.

Reply:
addDot=True is for value from metadata url. This is the fallback value and is used directly.

Reply:
It's a mistake and caught by the test.